### PR TITLE
Fix non-working link on video_player_styling_basics

### PR DESCRIPTION
**Pull request by chernodub**
#### Summary
Replaced broken link with a working one

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/guide/audio_and_video_delivery/video_player_styling_basics/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The markup for the custom controls now looks as follows:
 
 ### Related CSS alteration
 
-The previous article set the `display` property of the video controls to `block` in order to display them. This has now been changed to use a [`data-state` attribute](http://toddmotto.com/stop-toggling-classes-with-js-use-behavior-driven-dom-manipulation-with-data-states/), which this code already uses to handle its [fullscreen implementation](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player#fullscreen).
+The previous article set the `display` property of the video controls to `block` in order to display them. This has now been changed to use a [`data-state` attribute](https://ultimatecourses.com/blog/stop-toggling-classes-with-js-use-behaviour-driven-dom-manipulation-with-data-states), which this code already uses to handle its [fullscreen implementation](/en-US/docs/Web/Guide/Audio_and_video_delivery/cross_browser_video_player#fullscreen).
```
Comment (hamishwillee) [inline files/en-us/web/guide/audio_and_video_delivery/video_player_styling_basics/index.md:47]: Thanks very much - this is the same link redirected to another site, so it is certainly no worse than before.

Do you think that there is a good MDN link or a better external link that might be used instead (often articles are created and then later on we create better links)? I'm not saying there are but worth looking ...

For example 
- https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
- https://css-tricks.com/a-complete-guide-to-data-attributes/

**Review by hamishwillee — APPROVED** [review]
Approving, as better than having a broken link. Thanks @chernodub - if you do think there is an acceptable MDN link (as per https://github.com/mdn/content/pull/11581/files#r776579338 then feel free to add as a follow up PR).

**Comment by chernodub** [comment]
Hi @hamishwillee, sorry, I totally missed your comment and I will look for something better, and if I find anything I'll definitely propose it here. Thanks for your work! 🙌

**Comment by hamishwillee** [comment]
Thanks! Much appreciated, and no worries. Github gives me too many messages too ;-)

**Pull request merged**